src/group_theory/perm/cycles.lean

```diff
@@ -106,6 +109,20 @@ by classical; exact ⟨(n % order_of f).to_nat, by {
   have := n.mod_nonneg (int.coe_nat_ne_zero.mpr (ne_of_gt (order_of_pos f))),
   rwa [← gpow_coe_nat, int.to_nat_of_nonneg this, ← gpow_eq_mod_order_of] }⟩
 
+lemma is_cycle.exists_pow_ge_one_eq_one [fintype β] {f : perm β} (hf : is_cycle f) :
```

Comment:
I think you could probably drop the `ge_one` from the statement, since no other version of this statement would be interesting enough to exist.
```suggestion
lemma is_cycle.exists_pow_eq_one [fintype β] {f : perm β} (hf : is_cycle f) :
```

Reply:
Is this statement true as an `iff`?

Reply:
It is not true as an `iff` -- `(0 1 2) (3 4)` is not a cycle, but has order 6.